templates/raspbian_cattlepi/resources/usr/share/initramfs-tools/scripts/cattlepi-base/helpers

```diff
@@ -447,3 +447,22 @@ cattlepi_bring_up_wlan() {
         return 1
 	fi
 }
+
+cattlepi_sdmount() {
+    # Explicitly mount the sdcard partition to /boot
+    # We don't give the caller any other option as of right
+    # now since this partition doesn't need to be mounted
+    # anywhere else and doing so could be dangerous
+    echo "Mounting sdcard to /boot"
+    mkdir /boot
+    mount -n -t vfat /dev/mmcblk0p1 /boot
+}
+
+cattlepi_sdmount_RO() {
```

Comment:
probably need at cattlepi_umount function that can be invoked. 
also i would prefer cattlepi_sdmount_ro but don't have a very strong opinion on it

Reply:
👍